NitroxServer/Banning/IpBanning.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NitroxServer.Banning
+{
+    public static class IpBanning
```

Comment:
Maybe make a interface (with a shared method "IsBanned") for more banning options in the future and instantiate it with AutoFac (no static class)